packages/core/content-releases/admin/src/components/tests/ReleaseModal.test.tsx

```diff
@@ -67,4 +68,94 @@ describe('ReleaseModal', () => {
     await user.clear(inputElement);
     expect(dialogSaveButton).toBeDisabled();
   });
+
+  it('should show scheduled fields when selecting schedule release', async () => {
+    const handleCloseMocked = jest.fn();
+    render(
+      <ReleaseModal
+        handleClose={handleCloseMocked}
+        handleSubmit={jest.fn()}
+        initialValues={{ name: 'title', date: null, time: '', timezone: '', scheduledAt: null }}
+        isLoading={false}
+      />
+    );
+    const dialogContainer = screen.getByRole('dialog');
+    const scheduleReleaseCheck = within(dialogContainer).getByRole('checkbox', {
+      name: /schedule release/i,
+    });
+
+    // Schedule release checkbox is not checked and date field is not visible
+    expect(scheduleReleaseCheck).not.toBeChecked();
+    const date = within(dialogContainer).queryByRole('combobox', {
+      name: /date/i,
```

Comment:
is there a reason you're using Regex to find the labels? You could use the actual names?

Reply:
no, just a habit.